ui/components/role-select.js

```diff
@@ -76,11 +54,9 @@ export default function RoleSelect({
             ref={setPopperElement}
             style={styles.popper}
             {...attributes.popper}
-            className={`absolute z-10 w-48 ${
-              direction === 'right' ? '' : 'right-0'
-            } mt-2 overflow-auto rounded-md border  border-gray-200 bg-white text-left text-xs text-gray-800 shadow-lg shadow-gray-300/20 focus:outline-none`}
+            className='absolute z-[8] w-48 overflow-auto rounded-md border  border-gray-200 bg-white text-left text-xs text-gray-800 shadow-lg shadow-gray-300/20 focus:outline-none'
```

Comment:
Instead of `z-[8]` it would be better to keep our z indices to 10, 20, 30 etc – would `z-10` work here? or changing the "higher" item to `z-20`, etc?